org.nongnu.gsequencer.gsequencer.json

```diff
@@ -229,6 +229,7 @@
                 },               
 		{
 		    "type": "patch",
+                    "strip-components": 0,
```

Comment:
One of these days you'll configure your editors indent level :P

Reply:
sorry, I really love tabs and don't feel like defaulting emacs to use spaces. Just did M-x untabify ;)

Reply:
I'm confident that `emacs` has supported per-directory settings for at least a few decades. I already know the instant this is merged the file will just become a mess =(